src/main/java/com/puppycrawl/tools/checkstyle/meta/XmlMetaReader.java

```diff
@@ -1,258 +0,0 @@
-////////////////////////////////////////////////////////////////////////////////
-// checkstyle: Checks Java source code for adherence to a set of rules.
```

Comment:
Did you check plugins to use this class XmlMetaReader ?

Reply:
I double checked and found usage in eclipse plugin.

I will revert this changes and add some tests to cover it

Reply:
done